components/link/link-mixin.js

```diff
@@ -0,0 +1,60 @@
+export const LinkMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			/**
```

Comment:
This might open up a debate... lol... this is on the broader topic of how we want properties documented: my personal opinion (and I am totally open to change it ;), or just follow suit in keeping things consistent) is that such comments don't add much value and just make me scroll more in the IDE.  I know we added them at some point in Polymer components to get some automagical documentation on the iron-demo page, but I never found it useful.  I agree that having the documentation close to the code is valuable, but we're going to be documenting the properties separately in the readme anyway, so documentation in-code seems like inviting inconsistency and double docs.  My view is that the code (with good naming) is the best documentation, and comments are for the things that aren't obvious.  Again... I'm completely open to other ideas/thoughts on it... including generating documentation, etc.

Reply:
My opinion is this stuff is better in the readme. I found the polymer documentation not useful too.

Reply:
Agree as well.